lesson_29/script1.js

```diff
@@ -0,0 +1,10 @@
+let isImgZoommed = document.querySelector('.service-container');
+
+isImgZoommed.addEventListener('click', (event) => {
```

Comment:
you have pretty the same code in different files and functions

you can create a function where the name of class you can pass as a param `zoomed-img` or `big-img` and selector `.service-container` could also come like a param

so you will have one function and one file which handle different events

Reply:
the same .js file could be added to multiple html files